bundles/scripting/sightly/engine/src/main/java/org/apache/sling/scripting/sightly/impl/engine/extension/ResourceRuntimeExtension.java

```diff
@@ -229,16 +238,24 @@ private Object getAndRemoveOption(Map<String, Object> options, String property)
 
     private void includeResource(final Bindings bindings, PrintWriter out, String script, String dispatcherOptions, String resourceType) {
         if (StringUtils.isEmpty(script)) {
-            LOG.error("Script path cannot be empty");
+            LOG.error("Resource path cannot be empty");
```

Comment:
Can we rather throw an exception instead? That will also help with the new maven plugin.